toolchain/semantics/semantics_ir.h

```diff
@@ -82,6 +82,13 @@ class SemanticsIR {
   }
   auto Print(llvm::raw_ostream& out, bool include_builtins) const -> void;
 
+  // Adds an empty node block, returning an ID to reference it.
+  auto AddBlock() -> SemanticsNodeBlockId {
```

Comment:
Naming AddNodeBlock would mirror GetNodeBlock; also, note how I've been placing all the Add/Get functions next to each other...

Reply:
Done.